README.md

```diff
@@ -0,0 +1,164 @@
+        <img src="https://img.shields.io/lgtm/grade/java/g/MovingBlocks/Terasology.svg?logo=lgtm&logoWidth=18" alt="Java Grade" />
+    </a>
+</div>
+
+<h3 align="center"><b>
+    <a href="#community">Community</a> | 
+    <a href="#installation">Installation</a> | 
+    <a href="#development">Development</a>  | 
+    <a href="#license">License</a> 
+</b></h3>
+
+The _Terasology_ project was born from a Minecraft-inspired tech demo and is becoming a stable platform for various types of gameplay settings in a voxel world.
+The [creators and maintainers](https://github.com/MovingBlocks/Terasology/blob/develop/docs/Credits.md) are a diverse mix of software developers, designers, game testers, graphic artists, and musicians. We encourage others to join!
+We encourage contributions from anybody and try to keep a warm and friendly community and maintain a [code of conduct](docs/CODE_OF_CONDUCT.md).
+
+## Community
+
+If you want to get in contact with the **Terasology** community and the whole **MovingBlocks** team, you can easily connect with us, share your ideas, report and solve problems.
+We are present in nearly the complete round-up of social networks. Follow/friend us wherever you want, chat with us and tell the world.
+
+&nbsp;
+
+<p align="center">
+    <a title="Discord" href="https://discord.gg/terasology">
+        <img src="./docs/images/discord.png" width="48px"/>
+    </a>
+    &nbsp;&nbsp;&nbsp;&nbsp;
+    <a title="Twitter" href="https://twitter.com/Terasology">
+    <img src="./docs/images/twitter.png" width="48px"/>
+    </a>
+    &nbsp;&nbsp;&nbsp;&nbsp;
+    <a title="Facebook" href="https://www.facebook.com/Terasology">
+        <img src="./docs/images/facebook.png" width="48px"/>
+    </a>
+    &nbsp;&nbsp;&nbsp;&nbsp;
+    <a title="Reddit" href="http://www.reddit.com/r/Terasology">
+        <img src="./docs/images/reddit.png" width="48px"/>
+    </a>
+    &nbsp;&nbsp;&nbsp;&nbsp;
+    <a title="Youtube" href="https://www.youtube.com/user/blockmaniaTV">
+        <img src="./docs/images/youtube.png" width="48px"/>
+    </a>
+    &nbsp;&nbsp;&nbsp;&nbsp;
+    <a title="Patreon" href="https://www.patreon.com/Terasology">
+        <img src="./docs/images/patreon.jpg" width="48px"/>
+    </a>
+    &nbsp;&nbsp;&nbsp;&nbsp;
+    <a title="Terasology Forum" href="https://forum.terasology.org">
+        <img src="./docs/images/forum.png" width="48px"/>
+    </a>
+</p>
+
+
+## Installation
+
+<table>
+    <tr>
+        <td></td>
+        <th>Minimum Requirements</th>
+    </tr>
+    <tr>
+        <td>System (OS)</td>
+        <td>Windows, MacOS, Linux (64 bit)</td>
+    </tr>
+    <tr>
+        <td>Processor (CPU)</td>
+        <td>dual-core CPU</td>
+    </tr>
+    <tr>
+        <td>Memory (RAM)</td>
+        <td>2 GB</td>
+    </tr>
+    <tr>
+        <td>Graphics (GPU)</td>
+        <td style="vertical-align:top">
+            Intel HD Graphics (Gen 7)<br/>
```

Comment:
Maybe a comment to remind that if you have integrated graphics _and_ dedicated make sure your game doesn't end up stuck using this chip?

Reply:
done